theme/templates/includes/my-resources-trows.html

```diff
@@ -0,0 +1,109 @@
+{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
+{% for res in collection %}
+<tr class="data-row">
+    {# Selection controls #}
+    <td  data-col="actions">
+        <input class="row-selector" type="checkbox">
+        {# Delete resource forms #}
+        <form class="hidden-form" data-id="form-delete-{{ res.short_id }}"
+                data-form-type="delete-resource" method="POST"
+                action="/hsapi/_internal/{{ res.short_id }}/delete-resource/">
+            {% csrf_token %}
+        </form>
+        {% if res.is_favorite %}
+            <span data-form-id="form-favorite-{{ res.short_id }}"
+                    data-form-type="toggle-favorite"
+                    class="glyphicon glyphicon-star btn-inline-favorite isfavorite"></span>
+
+            <form class="hidden-form" data-id="form-favorite-{{ res.short_id }}"
+                    action="/hsapi/_internal/{{ res.short_id }}/label-resource-action/"
+                    method="POST">
+                {% csrf_token %}
+                <input type="hidden" name="action" value="DELETE">
+                <input type="hidden" name="label_type" value="FAVORITE">
+            </form>
+        {% else %}
+            <span data-form-id="form-favorite-{{ res.short_id }}"
+                    data-form-type="toggle-favorite"
+                    class="glyphicon glyphicon-star btn-inline-favorite"></span>
+
+            <form class="hidden-form" data-id="form-favorite-{{ res.short_id }}"
+                    action="/hsapi/_internal/{{ res.short_id }}/label-resource-action/"
+                    method="POST">
+                {% csrf_token %}
+                <input type="hidden" name="action" value="CREATE">
+                <input type="hidden" name="label_type" value="FAVORITE">
+            </form>
+        {% endif %}
+
+        <span class="glyphicon glyphicon-tag btn-inline-label" data-toggle="dropdown"
+                aria-expanded="false"></span>
+
+        <div class="dropdown-menu inline-dropdown" role="menu">
+            <div class="panel-body" role="form">
+                <ul data-resource-id="{{ res.short_id }}" class="list-group list-labels">
+                </ul>
+            </div>
+        </div>
+    </td>
+    {# Type #}
+    <td data-col="resource-type">
+        {% include "includes/res_type_col.html" with resource=res %}
+    </td>
+    {# Title #}
+    {% with metadata=res.metadata %}
+        <td>
+            <strong><a href="{{ res.get_absolute_url }}">{{ metadata.title|escape }}</a>
```

Comment:
`metadata.title` will generate a separate query for each resource. Use `res.title`